eng/testing/SetIcuMode.cmd

```diff
@@ -0,0 +1,40 @@
+:: This script is intended for our ICU tests
+:: the reason why we need to do this is because we build the tests
+:: in one version of Windows and then send the tests to helix to
+:: multiple Windows SKUs, including SKUs that doesn't contain ICU,
+:: since the globalization mode is calculated very early in the process
+:: we can't enable running ICU at all in this SKUs and then skip the tests
+:: at runtime, because xunit will start and then blow up when trying to load ICU.
+:: So with this script, we check if icu.dll is not present on the system, then we
+:: fallback to run on NLS mode.
+
+@echo off
+setlocal ENABLEEXTENSIONS
+setlocal ENABLEDELAYEDEXPANSION
+
+if EXIST %WINDIR%\system32\icu.dll (
+    echo ICU exists in system, running tests with ICU.
```

Comment:
Do we need to check for icuuc.dll or icuin.dll, or is that only relevant on a version of Win10 we don't support?

Reply:
Yeah, those only come into play in versions of Windows10 we don't support, so checking for icu.dll is enough.

Reply:
More info on these DLLs: https://docs.microsoft.com/windows/win32/intl/international-components-for-unicode--icu-#history-of-changes-to-the-icu-library-in-windows

Reply:
Ill add a comment to the script

Reply:
Sorry, didn't see your comment "we don't support it" before I left my follow-up. Please feel free to disregard this thread.